services/blockchain-indexer/shared/dataService/business/transactions.js

```diff
@@ -162,13 +165,16 @@ const getTransactions = async params => {
 				};
 			}
 
+			const [{ isFinal: isFinalInt }] = await blocksTable.find({ id: indexedTxInfo.blockID }, ['isFinal']);
+			const isFinal = isFinalInt === 1;
```

Comment:
It should already be returning a `boolean`!? Is it not?

Reply:
It stored as tinyInt in database.

Reply:
The `knex/mysql` libraries take care of the conversion of integer to a boolean. SQL DBs usually store booleans as an integer for space optimisation. As long as our schema defines the dataType of the property to be boolean, then we do not need this logic here.

Reply:
it is indeed defined `boolean` in the schema but we are getting integer value from the query for some reason.